### PR TITLE
Update to Dipy 1.2.0

### PR DESCRIPTION
**Pull request by 36000**
The load_tractogram in dipy v1.1.1 uses srow to get the affine information from a reference. However, some files have affine stored in the q info (q offset, pixdim), not s. This is fixed in the current dipy master, where they find the best source for affine information, but not dipy 1.1.1 .

Until a new version of dipy is released, we can use this PR with some datasets. This PR will probably not need to be merged, especially if a new version of dipy is releasing soon.

### CODE REVIEW

**Comment by 36000** [comment]
Fixes #263

**Comment by arokem** [comment]
Heads up that DIPY 1.2 is due to be released any day now. We can probably require that as a dependency and then merge this PR?

File: AFQ/segmentation.py [hunk 1]
```diff
@@ -387,9 +387,17 @@ def _check_sl_with_inclusion(self, sl, include_rois, tol):
         dist = []
         for roi in include_rois:
             # Use squared Euclidean distance, because it's faster:
-            dist.append(cdist(sl, roi, 'sqeuclidean'))
-            if np.min(dist[-1]) > tol:
-                # Too far from one of them:
+            try:
```
Comment (arokem) [inline AFQ/segmentation.py:390]: Maybe move this into a different PR? I think we should catch the case where the ROI is empty before we are here.
